test/attr-for-selected.html

```diff
@@ -210,18 +210,26 @@
       });
 
       test('selection is updated after fallback is set', function() {
-        selector.fallbackSolution = undefined;
+        selector.fallbackSelection= undefined;
         selector.select('non-existing-value');
```

Comment:
Hm this assertion throws an error, because the selected item is `items[2]`. Since fallback selection was set to default, default was selected before this test starts. Then we reset `fallback-selection`, which triggers `_updateSelected`. However, since `this.selected === 'default'`, no change is propagated. Then we selected `non-existing-value`, but since we reset `fallback-selection` the selected value is not altered.

I am not sure why I even put these undefined lines in the tests, should we just remove them? Or if we don't, then we should remove the attribute from the `test-fixture`.